library/psa_crypto_slot_management.h

```diff
@@ -203,12 +203,16 @@ psa_status_t psa_validate_key_location( psa_key_lifetime_t lifetime,
 /** Validate the persistence of a key.
  *
  * \param[in] lifetime  The key lifetime attribute.
+ * \param[in] key_id    The key identifier.
  *
  * \retval #PSA_SUCCESS
+ * \retval #PSA_ERROR_INVALID_HANDLE
```

Comment:
_Originally posted by @Summer-ARM in https://github.com/ARMmbed/mbedtls/pull/4189#discussion_r587332617_
> sorry, do I miss something? didn't see this return value in the function implementation.

Reply:
@Summer-ARM It looks like [`psa_validate_key_id()`](https://github.com/ARMmbed/mbedtls/pull/4198/files#diff-620cd14c340be363ec51956f5e66ec5233e9dffa97169020f0c3a0fd8d138359R387) is returning `PSA_ERROR_INVALID_HANDLE` instead of `PSA_ERROR_INVALID_ARGUMENT`, as it's documentation would suggest.

Reply:
There is indeed an issue here but not related to the issue this PR is solving. Thus not addressed in this PR. It is now addressed in #4279.